readthedocs/oauth/notifications.py

```diff
@@ -55,3 +55,22 @@ def get_context_data(self):
             ),
         })
         return context
+
+
+class SendBuildStatusFailureNotification(SiteNotification):
+
+    context_object_name = 'project'
+    failure_level = ERROR_PERSISTENT
```

Comment:
I don't know if we need it to persist. I don't know the best pattern for this, but we should probably err on the side of something that disappears, so we don't overwhelm users.

Reply:
@ericholscher `ERROR_PERSISTENT` Disappears when clicked on the link and error `ERROR_NON_PERSISTENT` disappears on page reload. which do you think we should have. and also both of then will re appear every time on failure. Do we want that or do we want something different?

Reply:
I think the one that disappers on page-reload makes sense. We can adjust over time.